src/flows/components/FlowPage.tsx

```diff
@@ -34,6 +36,12 @@ const FlowFromRoute = () => {
     change(id)
   }, [id, change])
 
+  useEffect(() => {
+    if (currentID !== null) {
+      event(`Notebook Accessed. NotebookID: ${currentID}`)
```

Comment:
Remove the notebook I'd stuff after the period or else we won't be able to group these. If you want to keep that granularity, move it into the tags parameter,  evt(name, { tag: tagval })

Reply:
makes sense.